### PR TITLE
tickets/DM-34740: Add check for recent start to polling portion of build-sciplatlab

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

File: pipelines/sqre/infra/build_sciplatlab.groovy [hunk 1]
```diff
@@ -92,6 +100,7 @@ notify.wrap {
           def wf = obj.workflow_runs[0]
           status = wf.status
           conclusion = wf.conclusion
+	  created_at = LocalDateTime.parse(wf.created_at,"2020-01-22T19:33:08Z")
```
Comment (ktlim) [inline pipelines/sqre/infra/build_sciplatlab.groovy:103]: Is the indent here correct?  And I think the example string should not be there; the optional second argument is a formatter.

**Review by ktlim — APPROVED** [review]
General logic looks good.  Expanding the window to be more than a second doesn't seem justified and could conceivably run into problems with both manually-triggered and Jenkins-triggered builds, so that part is good.  I think the actual use of the `parse()` function needs a bit of correction.

File: pipelines/sqre/infra/build_sciplatlab.groovy (hunk 1 above)
Comment (athornton) [reply pipelines/sqre/infra/build_sciplatlab.groovy:103]: Indent is probably wrong, but the second argument is needed, I think, in order to tell it what this string it's received from GH is gonna look like.

Comment (athornton) [reply pipelines/sqre/infra/build_sciplatlab.groovy]: As you say, it looks like it should be able to figure out for itself what format that date is.

**Pull request merged**